config.yml

```diff
@@ -1,5 +1,7 @@
 init:
   home: "$HOME/.squadapp"
+build:
+  ldflags: ["-X github.com/cosmosquad-labs/squad/x/farming/keeper.enableAdvanceEpoch=true"]
```

Comment:
It'd be better not to have this as default.

Reply:
As we discussed offline, I will go create `config-test.yml` that is used only for testing.